dftimewolf/lib/collectors/grr_base.py

```diff
@@ -89,8 +94,9 @@ def _WrapGRRRequestWithApproval(
         self.logger.info('No valid approval found: {0!s}'.format(exception))
         # If approval was already sent, just wait a bit more.
         if approval_sent:
-          self.logger.info('Approval not yet granted, waiting {0:d}s'.format(
-              self._CHECK_APPROVAL_INTERVAL_SEC))
+          self.logger.info('Approval not yet granted, waiting {0:d}s:\n{1:s}'.
```

Comment:
nit: the `\n` will break the log output - can you split this into two `logger.info` lines?

Reply:
Done.